### DISCUSSION TITLE
2020 年总结 · 渐入佳境 - 面向信仰编程 · /2020-summary

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#643] [Announcements]
https://draveness.me/2020-summary/

**Comment by zcong1993** [comment]
您太谦虚了。在这个博客学习到了很多东西，在这里感谢一下。按照你的标准，我工作了五年感觉一事无成，哈哈

**Comment by nemoTyrant** [comment]
博主很棒，对于自己感兴趣的领域研究都比较深入。期待看到更多的精彩文章。

**Comment by skyjerry** [comment]
非常优秀

**Comment by GavinCruise** [comment]
> @zcong1993
> 您太谦虚了。在这个博客学习到了很多东西，在这里感谢一下。按照你的标准，我工作了五年感觉一事无成，哈哈

我也是有同样感觉，作者三年经验，太厉害了。

**Comment by birdmanwings** [comment]
从博主这学习了很多，新的一年继续努力，感谢博主的分享

**Comment by draveness** [comment]
> @GavinCruise 
> > @zcong1993
> > 您太谦虚了。在这个博客学习到了很多东西，在这里感谢一下。按照你的标准，我工作了五年感觉一事无成，哈哈
> 
> 我也是有同样感觉，作者三年经验，太厉害了。

都三年多了，不过这不是写 2020 年总结么 ; )

**Comment by fkdtz** [comment]
「毕业的第三个年头」，博主太强了，另外博客上有书单推荐吗？

**Comment by draveness** [comment]
> @fkdtz
> 「毕业的第三个年头」，博主太强了，另外博客上有书单推荐吗？

可以考虑搞个书单

**Comment by mrdear** [comment]
非常优秀，文章条理性以及思考深度都很好，感谢无私地分享

**Comment by shenjing023** [comment]
德莱文真的强

**Comment by Huweicai** [comment]
前排围观

**Comment by CNife** [comment]
大佬 NB，感谢你一直以来的分享！

**Comment by pengpaige** [comment]
感谢博主的分享，看了这篇非常惊讶你能在一年做这么多事情。可以说一下你业余的时间，大概有多少比例投入上面列出的 KR 们了吗？

**Comment by yuchanns** [comment]
表面上左神是三年，但这只是毕业三年，而写博客已经七年了，我就这么安慰自己（狗头

**Comment by qiushido** [comment]
和博主毕业相差一年 差距可能已经不止三年，实在佩服。因为对kubernetes和GO相关感兴趣，非常幸运早早的就收藏了站点。博主写的内容很容易理解，也很有深度，非常感谢作者提供这么多高质量的文章，期待更多优秀的文章。

**Comment by draveness** [comment]
> @pengpaige 感谢博主的分享，看了这篇非常惊讶你能在一年做这么多事情。可以说一下你业余的时间，大概有多少比例投入上面列出的 KR 们了吗？

这个也太难分了吧，说实话没有统计过...

**Comment by chenyahui** [comment]
可以全年在家远程办公也是很舒服了~

**Comment by OhYee** [comment]
Go 语言设计与实现 啥时候能买，迫不及待😜

**Comment by cospotato** [comment]
和博主同年毕业，阅读了几十篇博主的文章之后，发现自己和博主的兴趣点几乎完全契合，有着强烈的共鸣，我甚至有一种看到了自己脑海里所幻想的完美自己的感觉，我几乎向身边的所有小伙伴推荐了博主的博客。无需多言，继续向博主学习

**Comment by draveness** [comment]
> @cospotato 和博主同年毕业，阅读了几十篇博主的文章之后，发现自己和博主的兴趣点几乎完全契合，有着强烈的共鸣，我甚至有一种看到了自己脑海里所幻想的完美自己的感觉，我几乎向身边的所有小伙伴推荐了博主的博客。无需多言，继续向博主学习

哈哈哈哈感谢啊

> @OhYee  Go 语言设计与实现 啥时候能买，迫不及待😜

估计得下半年吧，具体不知道

**Comment by hxzhouh** [comment]
优秀

**Comment by xgw** [comment]
老哥读了研还是本科毕业就出来工作了呀

**Comment by draveness** [comment]
> @xgw
> 老哥读了研还是本科毕业就出来工作了呀

本科毕业

**Comment by ichengzi** [comment]
毕业快4年了，和博主一比， 真的感觉是啥也没做。 前几年一直在windows上做.net的开发， 去年开始转java和linux，看了博主不少设计系列文章👍。

> 操作系统真的是每个合格的软件工程师必须深入理解的话题，很多现实场景中的问题都能够在操作系统中找到类似的场景

这个深有感触，系统调用， 零拷贝，非阻塞IO等等这些名词，没有OS的知识真的是理解困难，这里linux真是个大宝藏，windows虽然有最完善的文档，但是在具体代码面前，还是代码容易理解一些。

**Comment by jacktrane** [comment]
在同样的年纪里面，博主那么强，而我哭了

**Comment by wuqinqiang** [comment]
感觉两年多的我好垃圾😂

**Comment by wuqinqiang** [comment]
> 在同样的年纪里面，博主那么强，而我哭了

吴泽强？？ 我叫吴亲强。卧槽，缘分/。。。。。。

**Comment by sumaolin** [comment]
优秀的人是如此的优秀！高山仰止！

**Comment by jiekun** [comment]
给大佬点个赞~

**Comment by korilin** [comment]
大三生路过，《为什么这样设计》系列很多文章总能带给我很多自主思考，博主的文章除了所述知识还教会了我很多东西，😄

**Comment by draveness** [comment]
> 大三生路过，《为什么这样设计》系列很多文章总能带给我很多自主思考，博主的文章除了所述知识还教会了我很多东西，😄

需要注意的是作者写的东西不一定是对的，如果你觉得是错的，那可能就是错的，我们可以来讨论什么是对的

**Comment by korilin** [comment]
> @draveness
> > 大三生路过，《为什么这样设计》系列很多文章总能带给我很多自主思考，博主的文章除了所述知识还教会了我很多东西，😄
> 
> 需要注意的是作者写的东西不一定是对的，如果你觉得是错的，那可能就是错的，我们可以来讨论什么是对的

对的，我也有在写博客，之前和博主一开始一样也只是做做笔记，最近也开始按照自己所学知识写一些对外分享的。每次写博客和阅读别人的文章的时候总是会做延申思考，我是否真的掌握了这个内容，其中关联到的知识有哪些，文章所写和自己原本掌握的有什么不同，我的认知是对的还是文章所描述的是对的，为什么会导致某一方出现这个错误等等。有时候感觉阅读一篇好的博客或者自己踏实地写一篇博客，提升真的非常大

**Comment by Fwc1010** [comment]
希望大佬可以分享一下成长经历

**Comment by tuweiguang** [comment]
19年开始发现大佬,然后一直看大佬的输出,真的太强了

**Comment by yFeii** [comment]
hi，想问下 https://img.draveness.me/whys-the-design-traffic-2021-03-29-16169501059042.png 这个统计是接入第三方平台的数据统计么
袄，是 Google Analytics 么

**Comment by Dcell** [comment]
非常优秀，吾辈之楷模

**Comment by allbuleyu** [comment]
看大佬的博客半年多了,学到了好多东西,感谢你的输出,新书出了就去买.
希望大佬能出一份书单,很崇敬大佬,希望能和大佬一样强.

**Comment by MarkLux** [comment]
感谢总结和分享，十分受用！

**Comment by yzhaoyu** [comment]
感谢大佬的输出，学习了很多！希望以后能继续向你学习！

**Comment by visonforcoding** [comment]
吾辈之楷模

**Comment by wwulfric** [comment]
想知道这些研究和作者本职工作内容相差大么，是如何平衡的？

**Comment by YYYMagic** [comment]
在我入职后不久大佬便离职了，没能一睹真容，实在是吾辈学习的榜样。

**Comment by draveness** [comment]
> @wwulfric
> 想知道这些研究和作者本职工作内容相差大么，是如何平衡的？

多数还是相关的，感兴趣比较重要

> @YYYMagic
> 在我入职后不久大佬便离职了，没能一睹真容，实在是吾辈学习的榜样。

🤣

**Comment by Fwc1010** [comment]
> @YYYMagic
> 在我入职后不久大佬便离职了，没能一睹真容，实在是吾辈学习的榜样。

从字节走了？

**Comment by A1311981684** [comment]
啊这，同样是毕业三年。我和dalao差距是真的大。。。。

**Comment by gricn** [comment]
> @Stark-fwc
> > @YYYMagic
> > 在我入职后不久大佬便离职了，没能一睹真容，实在是吾辈学习的榜样。
> 
> 从字节走了？

去虾皮了

**Comment by zihengCat** [comment]
大佬您太谦虚啦，我这个刚毕业参加工作的菜鸟程序员，以德莱文大神您为榜样！^_^

**Comment by GuihaiHU** [comment]
博主的文章写得非常好

**Comment by HiAriesZhou** [comment]
没想到作者也是17年毕业，能在工作三年的节点，达到这样的专业素养，真心钦佩。
在获取、整理和输出知识的能力上，都值得同龄人学习。

**Comment by tabris233** [comment]
看了大佬的博客，时刻提醒我自己是个大废物。

大三拿到offer之后就听领导说的“傻傻的全身心投入工作”。毕业两年了会的全是拉通对齐，甩锅撕逼。技术上一直在倒退。

**Comment by okko123** [comment]
毕业工作快10年了。都没有达到博主毕业3年的水平实在惭愧。基本是会用，但不深入。有点浅尝即止

**Comment by lewki** [comment]
请教一下， 你整个网站用的什么模版构建出来的，非常喜欢这个布局风格， 学一个。

**Comment by liudongbo123** [comment]
“ 成为 Kubernetes 前 50 的提交者；” 请问大佬是怎么学习的，直接上手源码么，想请教一下学习方式&路线

**Comment by AustinWp** [comment]
大佬 给你膜拜

**Comment by grunmin** [comment]
看到三大支柱那里，想给博主推荐一个领域：认知科学

**Comment by numbernumberone** [comment]
想请教大佬一下，这个博客网站的前段时自己写的吗？还是用的什么开源组件吗？

**Comment by PeterlitsZo** [comment]
那 2022 年终总结呢？
